doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/coils.tex

```diff
@@ -1672,7 +1672,7 @@ \subsubsection{Standard Rating of Single-Speed DX Cooling Coils}\label{standard-
 Standard Ratings Reporting
 The values for these Standard Ratings are reported in the eplusout.eio output file (Ref. OutputDetailsAndExamples.pdf) and also in the predefined tabular output reports (Output:Table:SummaryReports object, '2017 Standard Ratings for DX Coils' and '2023 Standard Ratings for DX Coils'). 
 
-\subsubsection{Standard Rating Cooling Capacity}\label{standard-rating-cooling-capacity}
+\subsubsection{Standard Rating Cooling Capacity}
```

Comment:
Bunches of duplicate labels were simply deleted.  Not everything needs a label, we could probably remove all the unused ones and just have people label things *when they are needed*.